Lombiq.Tests.UI/Extensions/BasicOrchardFeaturesTestingUITestContextExtensions.cs

```diff
@@ -459,11 +485,21 @@ public static Task TestRegistrationAsync(this UITestContext context, UserRegistr
 
                 await context.TriggerAfterPageChangeEventAsync();
 
-                context.Scope.AtataContext.Go.ToNextWindow(new OrdinaryPage(pageTitle))
-                    .AggregateAssert(page => page
-                        .PageTitle.Should.Contain(pageTitle)
-                        .Find<H1<OrdinaryPage>>().Should.Equal(pageTitle))
-                    .CloseWindow();
+                var page = new OrdinaryPage(pageTitle);
+
+                context.Scope.AtataContext.Go.ToNextWindow(page);
+                page.PageTitle.Should.Contain(pageTitle);
+
+                if (customPageHeaderBy == null)
+                {
+                    page.Find<H1<OrdinaryPage>>().Should.Equal(pageTitle);
+                }
+                else
+                {
+                    context.Exists(customPageHeaderBy.Visible());
```

Comment:
Shouldn't we also check its content? To make it easier, we can just check if any of the inner text of the element contains the title, no stricter equality check necessary. Or: Perhaps better altogether to be able to provide a custom header checking logic altogether instead of just a `By`?

Reply:
There are some themes where the site name is not displayed and it's only in the `page.PageTitle` which we already check. So the first option is not a good solution.

The second one would be good I think. I will check it.